pkg/pb/synthetic_monitoring/checks_extra_test.go

```diff
@@ -212,6 +212,19 @@ func TestCheckType(t *testing.T) {
 			},
 			expected: CheckTypeTcp,
 		},
+		"traceroute": {
+			input: Check{
+				Target:    "127.0.0.1",
+				Job:       "job",
+				Frequency: 60000,
+				Timeout:   30000,
+				Probes:    []int64{1},
+				Settings: CheckSettings{
+					Traceroute: &TracerouteSettings{},
+				},
+			},
+			expected: CheckTypeTraceroute,
+		},
```

Comment:
Again could we move this either directly above or below Ping? So we can eyeball more easily that they are the same, or similar?